modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -1076,6 +1076,53 @@ describe('V2 Wallet:', function () {
     });
   });
 
+  describe('Algorand tests', () => {
+    let algoWallet: Wallet;
+
+    before(async () => {
+      // This is not a real THBAR wallet
+      const walletData = {
+        id: '650204cf43d8b40007cd9e11a872ce65',
+        coin: 'talgo',
+        keys: [
+          '650204b78a75c90007790bce979ae34d',
+          '650204b766c56a00072956c08fb9cdf1',
+          '650204b8ccf1370007b32bb8155dfbec',
+        ],
+        coinSpecific: {
+          rootAddress: '2ULRGE64U7LTMT5M6REB7ORHX5GLJYWHTIV5EAXVLWQTTATVJDGM5KJMII',
+        },
```

Comment:
Is it possible for a `talgo` wallet to also have a `baseAddress`?

Reply:
The consolidation handles this doble name for principal address In wallet platform too.

https://github.com/BitGo/bitgo-microservices/blob/b8f34891b5559c29806818564169881c1b8a6735/packages/wallet-platform/app/controllers/api/v2/consolidateAccountTS.ts#L305

Some coin has base address other root address. 
Algorand has rootAddress.